packages/triple-header/src/effects/rotate.tsx

```diff
@@ -0,0 +1,45 @@
+import { motion } from 'framer-motion'
+import { PropsWithChildren } from 'react'
+
+import { InitialEffectOptions } from './types'
+import { stringifyTransition } from './common'
+
+type ExtendedEffectOptions = InitialEffectOptions & {
+  degree: number
+}
+
+export interface RotateProps {
+  type: 'rotate'
+  options?: ExtendedEffectOptions
+}
+
+export default function Rotate({
+  children,
+  options: initialOptions,
```

Comment:
default value를 선언하면 아래 로직이 조금 간결해질 수 있지 않을까요

```suggestion
  options = {},
```